format_values.md

```diff
@@ -0,0 +1,90 @@
+# Format Values
+
+This document is a list of standardized values that can be used in certain fields in the format.
+
+## Frameworks
+
+The following is a list of standardized Framework names.
+
+- `framework-fabric-loader`
+- `framework-forge`
+- `framework-liteloader`
+- `framework-rift`
+- `framework-risugami-modloader`
+
+These names are prefixed with `framework-` to differentiate them between normal Projects. It also prevents possible collisions between Frameworks and other projects that have the same name.
+
+Hosts that serve Framework metadata MUST use these IDs. When a Framework ID is used in metadata, such as listed as a dependency, it MUST be one of the standardized values.
+
+---
+
+## Tags
+
+This is a list of standardized values that can be present in the `tags` field. It is not required to use these values, although it is encouraged to use them.
+
+- `mod`
+- `modpack`
+- `resourcepack`
+- `texturepack` (note: use `resourcepack` for packs that support it)
```

Comment:
What's the difference exactly?

Reply:
In 1.6 the official named changed from texture to resource. It exists in case you wish to specify *texture* pack rather than resource pack for whatever reason. `texturepack` could probably be removed though.